### ISSUE TITLE
Resolve Discrepancies Between ESRL data and @keeling_curve data

### ISSUE THREAD

**Issue by titojankowski** [giving-a-fuck-about-climate-change/carbon-doomsday#11]
I was reviewing existing CO2 measurement twitter handle @keeling_curve on twitter to compare to our dataset.

## Review of @Keeling_Curve Data
**1. ESRL Missing Data?**
@keeling_curve (https://twitter.com/Keeling_curve) has data for every single day for the past month. ESLR does not, ie 2017-7-7, 2015-7-8, 2017-06-15. 
_Is @keeling_curve getting different data?_

**2. @Keeling_Curve Data Different from ESRL?**
@keeling_curve data for 05/31/2017 (posted 11:37 AM - 1 Jun 2017) reads 409.59 ppm. Data from the ESLR NOAA site has 409.3, 409.65 ,409.52 (daily avg, monthly avg, weekly avg).
_Is @keeling_curve getting different data?_

**3. Data Freshness Review: @Keeling_Curve**
For the 10 days preceding 6/15/2017:
- **Stale**: @Keeling_Curve wasn't updated for 3 days June 16 - 19th. Wasn't updated for 2 days June 11th-13th
- **Uneven**: Post times for @Keeling_Curve vary widely, from 17.5 hours after EOD (data for 6/19/2017) to 59 hours after EOD (data for 6/17/2017).

## @Keeling_Curve data (2017-06-10 through 2017-06-19)
[text from tweet, posted time from tweet]
408.18 parts per million (ppm) CO2 in air 19-Jun-2017, posted 5:31 PM - 20 Jun 2017
**408.22 parts per million (ppm) CO2 in air 18-Jun-2017, posted 11:02 AM - 19 Jun 2017
408.81 parts per million (ppm) CO2 in air 17-Jun-2017, posted 11:02 AM - 19 Jun 2017
409.36 parts per million (ppm) CO2 in air 16-Jun-2017, posted 11:02 AM - 19 Jun 2017**
409.28 parts per million (ppm) CO2 in air 15-Jun-2017, posted 3:55 PM - 16 Jun 2017
409.58 parts per million (ppm) CO2 in air 14-Jun-2017, posted 12:04 PM - 15 Jun 2017
409.67 parts per million (ppm) CO2 in air 13-Jun-2017, posted 5:33 PM - 14 Jun 2017
**409.76 parts per million (ppm) CO2 in air 12-Jun-2017, posted 8:41 AM - 13 Jun 2017
409.71 parts per million (ppm) CO2 in air 11-Jun-2017, posted 8:41 AM - 13 Jun 2017**
409.21 parts per million (ppm) CO2 in air 10-Jun-2017, posted 8:32 PM - 11 Jun 2017

## ESRL data (2017-06-10 through 2017-06-19)
[Date,day,month,week]
2017-06-10,409.15,,409.65
2017-06-11,,,409.39
2017-06-12,409.75,,409.39
2017-06-13,409.62,,409.39
2017-06-14,409.44,,409.39
2017-06-15,,,409.39
2017-06-16,,,409.39
2017-06-17,408.74,,409.39
2017-06-18,408.67,,
2017-06-19,407.73,,

**Comment by titojankowski** [comment]
## @Keeling_Curve Source?
@Keeling_Curve data for today (408.18 ppm) matches the data shown on this chart: https://scripps.ucsd.edu/programs/keelingcurve/
But I have not yet found the dataset this comes from. The most "recent" datasets listed on these pages end in 2016 :\
http://scrippsco2.ucsd.edu/data/atmospheric_co2/primary_mlo_co2_record
http://scrippsco2.ucsd.edu/data/atmospheric_co2/mlo

## Intraday Data - 55 times a day!!
Good news is, the Scripps chart clearly shows intra-day data But, as above, I can't find the raw dataset for it (https://scripps.ucsd.edu/programs/keelingcurve/)

I counted the datapoints (pixellated) and estimate there are 55 datapoints each day, which is data about every 26 minutes.
![mlo_one_week](https://user-images.githubusercontent.com/129060/27385031-24c6f44a-565f-11e7-9150-3d209bd73b29.png)

@wickedwitch since you found that last great dataset from ESRL, thoughts on the Scripps data?

**Comment by lwm** [comment]
@titojankowski, nice investigation! Did you try to contact maintainers of these charts to get the location of these data sets? The intraday data would be particularly cool for 'zoom in' on day charts. I am certainly not against pulling from different data sources but we should be clear on the accuracy/benefits of each.

**Comment by titojankowski** [comment]
@lwm yes, I tweeted @keeling_curve and wrote their UCSD group email address as well.

Even without it I think we're all set for an MVP already. The existing daily ESRL dataset looks sufficient. All this stuff above seems "nice to have".

**Comment by lwm** [comment]
:+1: 

Sure, worth taking a look at all the same and nice to see it laid out here. Much appreciated.

**Comment by titojankowski** [comment]
==pasted from @lwm in gitter==

"Scripps reports daily averages based on local time in Hawaii, USA, where the monitoring station is located. NOAA reports daily averages based on local time in Boulder, Colorodo, USA where its headquarters is located for a global network of stations that monitor atmospheric levels for CO2 and other greenhouse gases." -> Might be the reason for differences
from here: https://www.co2.earth/daily-co2
